src/components/product/ProductSummary/product-summary.scss

```diff
@@ -0,0 +1,78 @@
+.product-summary[data-store-card] {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  max-width: 14rem;
+  height: 100%;
+  padding: var(--space-1);
+  background-color: var(--bg-neutral-lightest);
```

Comment:
Could you and this bg only on hover/focus? It would fix this bug with the shadow:
|Bug|
|-|
|![image](https://user-images.githubusercontent.com/11613011/147294229-cea07b91-1376-4eee-aa39-e0d8900fe8c3.png)|